pom.xml

```diff
@@ -1164,6 +1164,7 @@
             <exclude>**/dependency-reduced-pom.xml</exclude>
             <exclude>**/*.svg</exclude>
             <exclude>**/*.png</exclude>
+            <exclude>**/*.dita</exclude>
```

Comment:
Too wide, precise module and test dir please - if we put "main" dita we must set the header